packages/components/OptimizedImage.tsx

```diff
@@ -65,13 +64,18 @@ const transformURI = (
     return "";
   }
 
+  if (typeof uri !== "string") {
+    return uri;
+  }
+
   const isRelative = uri.startsWith("/");
   if (isRelative) {
     return uri;
   }
 
   // detect if raw CID
   try {
+    const { CID } = require("multiformats");
```

Comment:
how would this fix the issue?
I thought there is no react-native version of this library

Reply:
https://github.com/multiformats/js-multiformats/issues/276